cmd/launcher/options.go

```diff
@@ -187,6 +188,7 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		EnableInitialRunner:                *flInitialRunner,
 		EnrollSecret:                       *flEnrollSecret,
 		EnrollSecretPath:                   *flEnrollSecretPath,
+		AutoloadedExtensions:               strings.Split(*flAutoloadedExtensions, ","),
```

Comment:
Might be slightly cleaner to do this with the array style passing. See `flOsqueryFlags` for an example